pkg/client/selfhosted/selfhosted.go

```diff
@@ -31,6 +31,7 @@ const (
 
 	// HTTP headers to request API version
 	dockerAPIv1Header = "application/vnd.docker.distribution.manifest.v1+json"
+	ociV1Header       = "application/vnd.oci.image.manifest.v1+json"
```

Comment:
We need to be careful here also, as the responses could be different on OCI manifests too.. maybe we should include this in a separate pr incase if rollback.